mos-platform/cpm65/registers.S

```diff
@@ -0,0 +1,24 @@
+.macro register n, m
```

Comment:
Might want to add a comment on why this is used rather than the usual linker script mechanism. I assume it's something to do with the relocations?

Reply:
Comment added (it's so the linker can discard unused registers, which seems to happen a lot). (Especially in hand-written machine code.)